packages/payments-plugin/src/mollie/mollie.helpers.ts

```diff
@@ -67,6 +70,15 @@ export function toAmount(value: number, orderCurrency: string): Amount {
     };
 }
 
+/**
+ * Calculate tax amount per order line instead of per unit for Mollie
```

Comment:
I feel like we need a little mor explanation of the issue this is trying to solve for future reference. A link to issue #1939 and a brief note about why we can't rely on the Vendure lineTax value. Basically the example given in [this comment](https://github.com/vendure-ecommerce/vendure/issues/1939#issuecomment-1362962133) makes clear why we need this work-around.